### ISSUE TITLE
Print PR URLs after creation

### ISSUE THREAD

**Issue by guidodobboletta** [NerdWalletOSS/shepherd#53]
It would be nice to have an option to print the PR URL from the CLI after creation as some people usually copy paste the PR links to Slack to get them reviewed by someone else. This would make the process a lot easier.

**Comment by parshap** [comment]
Yes! I've wanted this too. In addition to printing the URLs of each, combined with #54 I'd also like to print a GitHub search url at the end that will show _all_ of the PRs for that migration.

Hopefully someone will have time soon to work on this. @guidodobboletta are you interested in attempting a PR for this at all? I'm happy to help if so.

**Comment by guidodobboletta** [comment]
I gave it a try but I'm not really familiar with Typescript/JS @parshap

**Comment by guidodobboletta** [comment]
Another enhancement that is pretty similar to this one: After running checkout print a summary of the repos that match so we don't have to check that one by one or resort to bash hacks @parshap
